hello_world/include/hello-world.hpp

```diff
@@ -2,9 +2,5 @@
 #include <iostream>
```

Comment:
I missed it the first time, but `iostream` should be in your source file too. You're only using `string` in the header.

Reply:
Since I'm no longer Printing shouldn't I just remove it?

Reply:
yeah. That's a better option.